### PR TITLE
Pending Tasks update for Cloudsiem

### PR DESCRIPTION
**Pull request by langmtt**

### CODE REVIEW

File: .taskcat.yml [hunk 1]
```diff
@@ -23,22 +24,23 @@ tests:
       Section1ToolingAndLoggingRegion: "us-east-2"
       Section1LogArchivingAccountId: ""
       Section1OrganizationRootID: ""
-      Section1DeliveryBucketPrefix: logs-delivery-$[taskcat_random-string]
+      Section1DeliveryBucketPrefix: logs-delivery
```
Comment (himanshu219) [inline .taskcat.yml:27]: previously I have seen that this fails without random string any reason why we are removing it.

File: templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-main.yaml [hunk 2]
```diff
@@ -78,10 +79,12 @@ Metadata:
       # Organization parameters
       ExecutionRoleRuleName:
         default: "Name of Execution Role for Organization"
-      CloudTrailRegions:
-        default: "CloudTrail Regions of Management Account"
-
-
+      CloudTrailRegion:
+        default: "CloudTrail Region of Management Account"
+      DelegatedAdminAccountId:
```
Comment (himanshu219) [inline templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-main.yaml:84]: Do we need to add delegated adminstrator for one region only in order to collect logs from multiple regions?

File: templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-main.yaml [hunk 3]
```diff
@@ -312,12 +331,12 @@ Resources:
...
       StackInstancesGroup:
         - Regions:
-            - !Ref CloudTrailRegions
+            - !Ref CloudTrailRegion
           DeploymentTargets:
             Accounts:
               - !Ref "AWS::AccountId"
       TemplateURL: !Sub
-        - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml'
+        - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}${QSVersion}/templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml'
```
Comment (himanshu219) [inline templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-main.yaml:339]: Fix the `AllowedPattern: "^[0-9a-zA-Z-/]*$"` for QSS3KeyPrefix such that it always ends with `/`

File: templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml [hunk 4]
```diff
@@ -27,6 +27,12 @@ Parameters:
     Description: CloudTrail name
     Type: String
 
+  pDelegatedAdminAccountId:
```
Comment (himanshu219) [inline templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml:30]: this will be log archival account id right ?

Comment (langmtt) [reply templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-org.yaml:30]: No, Delegated administrator account(Audit Account) has administrative permissions on all trails and event data stores in the organization

File: templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-main.yaml (hunk 2 above)
Comment (langmtt) [reply templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-main.yaml:84]: We should let the user specify a region to collect logs from multiple regions

File: templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-main.yaml (hunk 3 above)
Comment (langmtt) [reply templates/apps/sumo-aws-apps/cloudtrail-org/cloudtrail-main.yaml:339]: fixed

**Review by himanshu219 — APPROVED** [review]

**Pull request merged**